components/net/protocols/mod.rs

```diff
@@ -96,8 +104,18 @@ impl ProtocolRegistry {
     pub fn is_fetchable(&self, scheme: &str) -> bool {
         self.handlers
             .get(scheme)
-            .map(|handler| handler.is_fetchable())
-            .unwrap_or(false)
+            .is_some_and(|handler| handler.is_fetchable())
```

Comment:
Very minor, but this can possibly be made a bit simpler like this:

```suggestion
            .is_some_and(ProtocolHandler::is_fetchable)
```

Reply:
Seems like we can't do this since there's a box and we also need to do the deref